### ISSUE TITLE
Console app that contains a single command doesn't show the detail for help.

### ISSUE THREAD

**Issue by fog-bank** [Cysharp/ConsoleAppFramework#32]
When `help [command]` is executed for an app that contains a single command with `CommandAttribute`, the app shows the command list rather than the detail of that command.

### Steps to reproduce
1. Create .NET Core Console App named ConsoleApp1 (TargetFramework = netcoreapp2.1; LangVersion = latest)
2. Install ConsoleAppFramework 2.0.2
3. `dotnet build` and `dotnet run -p ConsoleApp1 -- help escape`

```c#
using System;
using System.Threading.Tasks;
using ConsoleAppFramework;
using Microsoft.Extensions.Hosting;
namespace ConsoleApp1
{
    class Program : ConsoleAppBase
    {
        static async Task Main(string[] args)
        {
            await Host.CreateDefaultBuilder().RunConsoleAppFrameworkAsync<Program>(args);
        }
        [Command("escape")]
        public void UrlEscape([Option("i", "URL")] string input = "")
        {
            Console.WriteLine(Uri.EscapeDataString(input));
        }
    }
}
```
### Expected
```
Usage: ConsoleApp1 escape [options...]

Options:
  -i, -input <String>    URL (Default: )
```

### Actual
```
Usage: ConsoleApp1 <Command>

Commands:
  escape     
```

### Workarounds
* Don't use `CommandAttribute` for a single command.
* Add any command.

### Environment
* Windows 10 v1809
* Visual Studio 2017.9
* .NET Core SDK 2.1.511
* .NET Core runtime 2.1.15

**Comment by mayuki** [comment]
Sorry for late reply. We fixed this issue and just released v2.1.0.

**Comment by fog-bank** [comment]
Thank you for handling this issue.
